app/models/organization.rb

```diff
@@ -39,6 +40,7 @@ class Organization < ApplicationRecord
   validate :deadline_after_reminder
 
   belongs_to :account_request, optional: true
+  belongs_to :ndbn_member, class_name: 'NDBNMember', optional: true
```

Comment:
I think we need to change this to:
`belongs_to :ndbn_member, class_name: 'NDBNMember', foreign_key: :ndbn_member_id, optional: true` for the association test to run.